Adyen/Model/TerminalApi/EventToNotifyType.cs

```diff
@@ -1,7 +1,6 @@
 ﻿namespace Adyen.Model.TerminalApi
 {
     /// <remarks/>
-    [System.CodeDom.Compiler.GeneratedCodeAttribute("xsd", "4.6.1055.0")]
```

Comment:
Q: This class used to be generated, we're now removing the tag and made manual changes to this event type. Do you happen to have the exhaustive list of attributes for `EventToNotifyType`?

Reply:
https://docs.adyen.com/point-of-sale/design-your-integration/notifications/event-notifications/#event-notification-types